lms/templates/discussion/_js_head_dependencies.html

```diff
@@ -1,14 +1,8 @@
 <%namespace name='static' file='../static_content.html'/>
 
-<script type="text/javascript" src="${static.url('js/split.js')}"></script>
-<script type="text/javascript" src="${static.url('js/vendor/jquery.truncate.js')}"></script>
-<script type="text/javascript" src="${static.url('js/jquery.ajaxfileupload.js')}"></script>
-<script type="text/javascript" src="${static.url('js/Markdown.Converter.js')}"></script>
-<script type="text/javascript" src="${static.url('js/Markdown.Sanitizer.js')}"></script>
-<script type="text/javascript" src="${static.url('js/Markdown.Editor.js')}"></script>
 <script type="text/javascript" src="${static.url('js/jquery.autocomplete.js')}"></script>
```

Comment:
I don't see any usages of Autocompleter in the code base. I wonder if this is just not used anymore, or if it is required by one of the Markdown libraries/